Description: Upstream fix to Makefile.am
 This patch disables the use of m4/ which doesn't exist in upstream's tarball
Author: Roland Stigge <stigge@antcom.de>

--- pstoedit-3.51.orig/Makefile.am
+++ pstoedit-3.51/Makefile.am
@@ -1,4 +1,3 @@
-ACLOCAL_AMFLAGS = -I m4
 EXTRA_DIST=copying autogen.sh readme.install
 AUTOMAKE_OPTIONS = foreign
 SUBDIRS = config doc examples contrib misc os2 othersrc src msdev 
